src/lwc/rd2EntryForm/rd2EntryForm.html

```diff
@@ -1,176 +1,197 @@
 <template>
-    <header class="slds-modal__header slds-clearfix">
-        <h2
-            class="slds-text-heading_medium slds-text-heading--medium slds-hyphenate header-string slds-p-top_large slds-p-horizontal_medium">
-            {header}
-        </h2>
-    </header>
-
-    <lightning-record-edit-form record-id={recordId} object-api-name="npe03__Recurring_Donation__c"
-        onsubmit={handleSubmit} onerror={handleSaveError} onsuccess={handleSuccess} data-id="outerRecordEditForm">
-
-        <div class="slds-modal__content slds-p-top_none slds-p-horizontal_medium slds-p-bottom_medium">
-            <template if:true={isLoading}>
-                <div class="slds-grid slds-grid_align-center">
-                    <div class="slds-is-relative slds-size_1-of-1">
-                        <lightning-spinner alternative-text={customLabels.spinnerAltText} data-text={loadingText}
-                            class="loading-spinner">
-                        </lightning-spinner>
-                        <c-util-stencil row-count="20" column-count="2"></c-util-stencil>
+    <section
+        role="dialog"
+        tabindex="-1"
+        aria-labelledby="modal-heading-01"
+        aria-modal="true"
+        aria-describedby="modal-content-id-1"
+        class="slds-modal slds-fade-in-open"
+        onkeyup={handleKeyUp}
+    >
+        <div class="slds-modal__container">
+            <header class="slds-modal__header slds-clearfix">
+                <lightning-button-icon
+                                class="slds-modal__close"
```

Comment:
Let's move indentation to be as for other elements ( one tab instead of three in this case).